src/coreclr/vm/eventtrace.cpp

```diff
@@ -7636,6 +7636,8 @@ void ETW::CompilationLog::TieredCompilation::GetSettings(UINT32 *flagsRef)
         None = 0x0,
         QuickJit = 0x1,
         QuickJitForLoops = 0x2,
+        TieredPGO = 0x3,
+        ReadyToRun = 0x4,
```

Comment:
These should be separate bits so that multiple can be set:
```suggestion
        TieredPGO = 0x4,
        ReadyToRun = 0x8,
```

Probably should also add the `[Flags]` attribute to the enum. Does PerfView show the names correctly when multiple flags are set?